UI-Session-4-Assignment/index.html

```diff
@@ -0,0 +1,196 @@
+    </header>
+
+    <!-- cover section containing Download and feature option -->
+    <section id="cover-section">
+        <div>
+            <h1>Get the most fun weather app</h1>
+            <a href="" id="download_button">Download</a>
+            <a href="" id="features_button">Features</a>
+        </div>
+        <div>
+            <img src="images\phone1.png" alt="Cloudy image">
+        </div>
+    </section>
+
+    <section id="container">
+        <!--Feature section containing features of  -->
+        <section id="feature-section">
+            <h1 class="heading">Perfect Features</h1>
+            <h5 class="sub-heading">Only necessary</h5>
+            <section id="features-headline">
+                <!-- feature 1-->
+                <div class="feature" id="feature1">
+                    <img src="images\feature1.png" alt="Usability image">
+                    <div>
+                        <h1>Usability</h1>
+
+                        <h5>Sometimes the simplest things are the hardest to find. So we created a nwe line for everyday life.</h5>
+                    </div>
+                </div>
+                <!-- feature 2-->
+                <div class="feature" id="feature2">
+                    <img src="images\feature2.png" alt="Parallex Effect image">
+                    <div>
+                        <h1>Parallex Effect</h1>
+
+                        <h5>Sometimes the simplest things are the hardest to find. So we created a nwe line for everyday life.</h5>
+                    </div>
+                </div>
+                <!-- feature 3-->
+                <div class="feature" id="feature3">
+                    <img src="images\feature3.png" alt="Unlimited Colors image">
+                    <div>
+                        <h1>Unlimited Colors </h1>
+
+                        <h5>Sometimes the simplest things are the hardest to find. So we created a nwe line for everyday life.</h5>
+                    </div>
+                </div>
+            </section>
+            <!-- feature images-slider-->
+            <section id="feature-image-slider">
+                <img id="slide-1-image" src="images\drag1.png" alt="Moving images">
+                <img id="slide-2-image" src="images\drag2.png" alt="Moving images">
+                <img id="slide-3-image" src="images\drag3.png" alt="Moving images">
+                <img id="slide-4-image" src="images\drag1.png" alt="Moving images">
+                <img id="slide-5-image" src="images\drag2.png" alt="Moving images">
+            </section>
+        </section>
+
+        <!-- Screenshot-section containing screenshot of -->
+        <section id="Screenshot-section">
+            <h1 class="heading">Screenshots</h1>
+            <h5 class="sub-heading">The brighest images</h5>
+            <!-- screenshot 1-->
+            <div id="screenshot1" class="screenshots-content">
+                <img src="images\screen01.png" alt="Cloudy image">
+                <div>
```

Comment:
temperature and weather type images are not visible according to the layout.

Reply:
Need explaination